ibridges/session.py

```diff
@@ -40,11 +41,14 @@ def __init__(self, irods_env: dict = None, irods_env_path: str = None,
             if os.path.isfile(irods_env_path):
                 with open(irods_env_path, "r", encoding="utf-8") as f:
                     irods_env = json.load(f)
+                if not isinstance(irods_env, dict):
+                    raise ValueError(f"Error reading environment file '{irods_env_path}': "
```

Comment:
Maybe a TypeError is more approproate? And I think `{irods_env}` should be `{type(irods_env)}`.

Reply:
I honestly don't know. The reason I went for a value error instead of a type error was because the value comes from the file. The most common way to see this error is if they load a different kind of file. Anyway, I'll change it.